alerting/src/main/kotlin/org/opensearch/alerting/transport/TransportSearchMonitorAction.kt

```diff
@@ -109,7 +111,9 @@ class TransportSearchMonitorAction @Inject constructor(
                 }
 
                 override fun onFailure(t: Exception) {
-                    actionListener.onFailure(AlertingException.wrap(t))
+                    actionListener.onFailure(
+                        AlertingException.wrap(OpenSearchStatusException("Monitor not found.", RestStatus.NOT_FOUND))
```

Comment:
Issue #1057 expects that we check if this failure is caused by IndexNotFoundException  (could be nested) and return an empty response